src/openMVG/localization/LocalizationResult_test.cpp

```diff
@@ -48,8 +48,12 @@ localization::LocalizationResult generateRandomResult(std::size_t numPts)
   
   // random valid
   const bool valid = (numInliers % 2 == 0);
-  //
-  return localization::LocalizationResult(data, indMatch3D2D, pose, intrinsics, valid);
+
+  std::vector<voctree::DocMatch> matchedImages;
+  matchedImages.push_back(voctree::DocMatch(2, 0.5));
+  matchedImages.push_back(voctree::DocMatch(3, 0.8));
+
+  return localization::LocalizationResult(data, indMatch3D2D, pose, intrinsics, matchedImages, valid);
```

Comment:
need to add in the 2 tests the comparison between the values

in `LoadSaveBinSingle`
```c++
  // same matchedImages
  EXPECT_TRUE(res.getMatchedImages().size() == check.getMatchedImages().size());
  const auto matchedImagesGT = res.getMatchedImages();
  const auto matchedImages = check.getMatchedImages();
  for(std::size_t i = 0; i < res.getMatchedImages().size(); ++i)
  {
    EXPECT_TRUE(matchedImagesGT[i] == matchedImages[i]);
  }
```
and the same cut and paste in `LoadSaveBinVector `